crates/autopilot/src/infra/solvers/dto/solve.rs

```diff
@@ -132,7 +132,65 @@ impl Solution {
 }
 
 #[serde_as]
-#[derive(Clone, Debug, Default, Deserialize)]
+#[derive(Clone, Debug, Deserialize)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[serde(untagged)]
+pub enum TradedOrder {
```

Comment:
enum, so that we can give external solvers running non-reference driver enough time to adapt.

Reply:
I would respectfully disagree. The simplicity to use a single format for all orders beats the redundancy argument in this case. Especially since on both sides (autopilot and driver) we don't profit from this differentiation (we don't use it for anything) and have to do manual conversions.